assets/scss/core/_base.scss

```diff
@@ -68,10 +66,88 @@ textarea {
   border-radius: $dp-border-radius;
   font-size: $dp-font-size-form;
   border: 1px solid $dp-color-sylver;
+
+  &::placeholder {
+    color: $dp-color-default;
+    font-style: italic;
+  }
+
+  :focus {
+    border: 1px solid $dp-color-yellow;
+  }
+}
+/* validate form*/
+.field-group {
+  position: relative;
+  display: inline-block;
+  text-align: left;
+  width: 100%;
+  margin-bottom: 30px;
```

Comment:
Nacho told me that you are going to add the space lvls in another PR right?, maybe this line should be removed

Reply:
As we have already talked with @mvillaseco, he will add in this same PR.

Basically, constants to be used here, and atomic classes for other cases.